src/logic/operate.js

```diff
@@ -12,7 +16,7 @@ function operate(numberOne, numberTwo, operation) {
     case '-':
       result = num1.minus(num2);
       break;
-    case 'X':
+    case 'x':
       result = num1.times(num2);
       break;
     case '÷':
```

Comment:
As mentioned by the previous reviewer;

Division by zero is not handled properly yet. Let's fix this.
The reason is that when a division by zero happens, we should return an error message like `division error`

Divide a number by zero (e.g. 4 / 0)